src/demo/index.php

```diff
@@ -103,6 +103,9 @@ function gtag() {
                     <option>false</option>
                     <option>true</option>
                 </select>
+                
+                <label for="border_radius">Border Radius</label>
+                <input class="param" type="number" id="border_radius" name="border_radius" placeholder="4.5 default, 0 none, 0-xxx" pattern="[0-9]" title="Only Numbers">
```

Comment:
It's a number type input, so it shouldn't need these other attributes.

```suggestion
                <input class="param" type="number" id="border_radius" name="border_radius" placeholder="4.5">
```

There's also a JavaScript file where it should include the default value for border_radius so that it's only included when not the default.